optimize/client/src/components/Analysis/TaskAnalysis/OutlierDetailsTable.test.tsx

```diff
@@ -36,20 +37,23 @@ it('should render the table properly', () => {
   const tableBody = node.find('Table').prop<(string | JSX.Element)[][]>('body');
 
   expect(tableBody.length).toBe(2);
-  expect(tableBody[0]?.length).toBe(6);
+  expect(tableBody[0]?.length).toBe(5);
 
   // task 1 with name and variable
   expect(tableBody[0]?.[0]).toBe('Task 1');
   expect(tableBody[0]?.[1]).toBe('10');
   expect(tableBody[0]?.[2]).toBe('1 instance took 50% longer than average.');
-  const task1Variabes = shallow(tableBody[0]?.[3] as JSX.Element);
-  expect(task1Variabes.text()).toBe('variable1=true');
+  const task1ViewDetails = shallow(tableBody[0]?.[3] as JSX.Element);
+  expect(task1ViewDetails.text()).toBe('View details');
+  expect(task1ViewDetails.type()).toBe(ButtonBase);
```

Comment:
Let's avoid using the Carbon internals here.
We do the following instead to prevent the shallow from rendering the Button internals:
```
  const task1ViewDetails = shallow(<div>{tableBody[0]?.[3]}</div>);

  expect(task1ViewDetails.find(Button).text()).toBe('View details');
  expect(task1ViewDetails.find(Button)).toExist();
```